src/app/activity/activity.service.ts

```diff
@@ -25,6 +25,9 @@ export interface Task {
   feedbackReviewed?: boolean;
   loadingStatus?: boolean;
   isForTeam?: boolean;
+  dueDate?: string;
+  isOverdue?: boolean;
+  iconClass?: string;
```

Comment:
change this to `isDueToday?: boolean`
We shouldn't put html class name inside files other than HTML/CSS file